### PR TITLE
Create seminar series data table component

### LINKED ISSUE #1298: Create a table component for events
On the qiskit seminar series page, we need to show the past and upcoming events on separate tables. This issue consists of creating a general component for that. 

Requirements:
- [x] Use the [carbon design table component](https://www.carbondesignsystem.com/components/data-table/usage/)
- [x] The styles should fit the [Figma design](https://www.figma.com/file/nqFBhDOUq6Dknbw9pM92AE/1263-individual-event-pages?node-id=122%3A3274)
- [x] The component should receive the data as props

### PR DESCRIPTION
**Pull request by y4izus**
Fix #1298 

To see the component go to /events/seminar-series

Note: this PR just create the table and a section to see it. It doesn't have the final information on it

### CODE REVIEW

**Review by techtolentino — CHANGES_REQUESTED** [review]
@y4izus - nice work with implementing the table!

One comment regarding cell widths, on mobile.

In the design for mobile, the cell widths give a little more room for the cell data to display... 

Screen Shot 2021-01-14 at 10 07 19 AM


In this PR, the _mobile_ table looks like so:
Screen Shot 2021-01-14 at 10 10 42 AM



In the design, it looks like cell widths are based on column widths... if we can use that, that would be awesome. If not, I think we'd still provide great UX if we could designate a width or even a min-width, so that the cell data is more readable.


I also realize that this could happen in a follow-up PR as well (if we all agree)

Thoughts?

**Comment by techtolentino** [comment]
Last question - are you seeing these warnings in the console? Or is this just my local...

File: components/events/single-events/SeminarSeriesDataTable.vue [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  institution: string,
+  dateOfTalk: string,
+  linkToTalk: GeneralLink
+}
+
+@Component
+export default class SeminarSeriesDataTable extends Vue {
+  @Prop(Array) data!: tableRow[]
+
+  columns = ['Speaker', 'Name of talk', 'Institution', 'Date of talk', 'Link to talk']
```
[the remark this answers is not in the document]
Comment (y4izus) [reply components/events/single-events/SeminarSeriesDataTable.vue:35]: Sure!

**Comment by y4izus** [comment]
I can see your image with the warnings on the console, but I will take a look ;).

Related to the columns' width, I just use the carbon design component and didn't look at that (honestly). I'll try to improve it

**Review by eddybrando — APPROVED** [review]
Awesome @y4izus thanks a lot!

**Pull request merged**